src/Microsoft.Diagnostics.DebugServices.Implementation/Host.cs

```diff
@@ -103,6 +104,16 @@ public string GetTempDirectory()
 
         #endregion
 
+        #region ISettingsService
+
+        public virtual bool DacSignatureVerificationEnabled { get; set; }
```

Comment:
Doesn't this mean that dac validation can only happen at the host and not runtime level? Say I want to validate framework still.

Reply:
This change doesn't change the scope of this property. It has always been global just like the windbg property we use on windbg.